### PR TITLE
APPINT-33762: Cleanup beans dependencies related migration tasks

### PR DESCRIPTION
**Pull request by vadym-drokov**

### CODE REVIEW

File: main/plugins/org.talend.camel.designer/plugin.xml [hunk 1]
```diff
@@ -329,6 +338,15 @@
                 name="UpdateBeansLibrariesMigrationTask 7.3.1"
                 version="7.3.1">
           </projecttask>
+          <projecttask
+                beforeLogon="false"
+                breaks="7.3.0"
+                class="org.talend.camel.designer.migration.RemoveBeansDuplicateLibrariesMigrationTask"
+                description="Add Beans default libraries"
+                id="org.talend.camel.designer.migration.RemoveBeansDuplicateLibrariesMigrationTask.7.4.1"
```
Comment (dsergent) [inline main/plugins/org.talend.camel.designer/plugin.xml:346]: HI @vadym-drokov. It seems you declared twice the same migraiton task (line 301 and here).

File: main/plugins/org.talend.camel.designer/plugin.xml [hunk 2]
```diff
@@ -329,6 +338,15 @@
...
+                name="RemoveBeansDuplicateLibrariesMigrationTask 7.4.1"
+                version="7.4.1">
```
Comment (dsergent) [inline main/plugins/org.talend.camel.designer/plugin.xml:348]: Is it normal to use 7.4.1 as version parameter? As this version will never exist, 8.0.1 replaced 7.4.1.

**Review by apoltavtsev — APPROVED** [review]
LGTM

**Review by dsergent — APPROVED** [review]
LGTM

**Pull request merged**